web-react-ts/src/pages/services/graphs/TrendsMenu.tsx

```diff
@@ -37,7 +37,7 @@ const TrendsMenu = () => {
             color="quick-facts"
             onClick={() =>
               navigate(
-                `/quick-facts/this-month/${church.__typename.toLowerCase()}`
+                `/quick-facts/this-month/${church?.__typename.toLowerCase()}`
```

Comment:
If there is no church.typename, the navlink will lead to a 404. Are you able to understand why there is no church typename?